app/plugin_api_test.go

```diff
@@ -530,7 +530,6 @@ func TestPluginAPIGetTeamIcon(t *testing.T) {
 	th := Setup().InitBasic()
 	defer th.TearDown()
 	api := th.SetupPluginAPI()
-
```

Comment:
Unnecesary change to a not related file here

Reply:
That was unintended. Reverted it back.